ociinstaller/tempdir.go

```diff
@@ -7,23 +7,22 @@ import (
 	"strings"
 
 	"github.com/google/uuid"
-	"github.com/turbot/steampipe/filepaths"
 	"github.com/turbot/steampipe/utils"
 )
 
 type tempDir struct {
 	Path string
 }
 
-// NewTempDir :: returns the temp directory, creating it if it does not exist
-func NewTempDir(path string) *tempDir {
+// NewTempDir creates a directory under the given parent directory.
+func NewTempDir(parent string) *tempDir {
```

Comment:
what if creation fails?

Reply:
panics if creation fails - this should get floated up and handled in a `cmd` `defer/recover`

Reply:
https://github.com/turbot/steampipe/pull/1548/files#diff-c0b66fded0546d90b52decdba1957b2c7d72d2eeba43eab3c7e951b1fc79c763R29

Reply:
have you tested this? by forcing a panic...

I can't see what the link is showing